src/register/RegisterBase.cpp

```diff
@@ -26,7 +28,7 @@ RegisterBase::RegisterBase(char const * name,
   [name]() -> uavcan_register_Name_1_0
   {
     uavcan_register_Name_1_0 n;
-    n.name.count = std::min(strlen(name), uavcan_register_Name_1_0_name_ARRAY_CAPACITY_);
+    n.name.count = std::min(static_cast<size_t>(strlen(name)), static_cast<size_t>(uavcan_register_Name_1_0_name_ARRAY_CAPACITY_));
```

Comment:
```suggestion
    n.name.count = std::min(std::strlen(name), static_cast<size_t>(uavcan_register_Name_1_0_name_ARRAY_CAPACITY_));
```

redundant cast

Reply:
Sadly, this cast is not redundant in my build.